js/ui/text_box/texteditor_button_collection/custom.js

```diff
@@ -30,7 +30,8 @@ export default class CustomButton extends TextEditorButton {
         this._addToContainer($element);
 
         const instance = editor._createComponent($element, Button, extend({}, this.options, {
-            disabled: this._isDisabled()
+            disabled: this._isDisabled(),
+            defaultTemplatesMap: { content: "customButtonContent" }
```

Comment:
It's better to approve this name with a techwriter if we are going to publish it. Or we could use an array of template names which should not be looked up in the `integrationOptions.temapltes` instead.

Reply:
@AlexJameson agree with me

Reply:
I am doubted about the `custom` word. Any template which customer sets is custom. But we do not use this prefix for other template names. Why we use this word here?